### PR TITLE
example_configs: Update Radicale guide with configuration for Radicale 3.2.4

### PR DESCRIPTION
**Pull request by poperigby**
3.2.4 hasn't actually released yet, so maybe wait for this to be merged.

### CODE REVIEW

**Review by nitnelave — CHANGES_REQUESTED** [review]
You could maybe merge it as another paragraph: 
- config with plugin
- config with 2.3.4

That way if someone doesn't want to upgrade, they can still see the config for the plugin. And we can merge right now without waiting for the release.

File: example_configs/radicale.md
Comment (nitnelave) [inline example_configs/radicale.md:8]: Let's keep it as ldap://lldap:3890

**Comment by poperigby** [comment]
@nitnelave Should be good now.

**Review by nitnelave — APPROVED** [review]
Thanks a lot!

**Comment by poperigby** [comment]
No problem 🙂

**Pull request merged**
